UnboundLib/UnboundLib.csproj

```diff
@@ -35,9 +35,10 @@
   </PropertyGroup>
   
   <PropertyGroup>
-    <RoundsFolder>C:\Program Files (x86)\Steam\steamapps\common\ROUNDS</RoundsFolder>
+    <RoundsFolder>A:\Program Files\SteamLibrary\steamapps\common\ROUNDS</RoundsFolder>
```

Comment:
Don't change the value here. Copy `UnboundLib.csproj.user.dist` to `UnboundLib.csproj.user` and change the folder there. The user file doesn't end up in git, so you can make personal configs there.